cabot/settings.py

```diff
@@ -4,6 +4,8 @@
 from cabot.celeryconfig import *
 from cabot.cabot_config import *
 import xmlrunner
+import pymysql
+pymysql.install_as_MySQLdb()
```

Comment:
Where would you recommend? I tried manage.py earlier, but we use django-admin.py instead in production (which is not in the repo). Only other place I could think of where this would always be run before we actually try to connect to the db would be cabot_config.py...